src/test/java/com/microsoft/sqlserver/jdbc/bulkCopy/BulkCopyRowSetTest.java

```diff
@@ -42,41 +47,40 @@ public void testBulkCopyFloatRowSet() throws SQLException {
             rsmd.setColumnName(2, "c2");
             rsmd.setColumnType(1, java.sql.Types.FLOAT);
             rsmd.setColumnType(2, java.sql.Types.FLOAT);
-            
+
             Float floatData = RandomData.generateReal(false);
-            
+
             crs.setMetaData(rsmd);
             crs.moveToInsertRow();
             crs.updateFloat(1, floatData);
             crs.updateFloat(2, floatData);
             crs.insertRow();
             crs.moveToCurrentRow();
-            
+
             try (SQLServerBulkCopy bcOperation = new SQLServerBulkCopy(con)) {
                 bcOperation.setDestinationTableName(tableName);
                 bcOperation.writeToServer(crs);
             }
-            
-            ResultSet rs = stmt.executeQuery("select * from " + tableName);
-            rs.next();
-            assertEquals(floatData, (Float) rs.getFloat(1));
-            assertEquals(floatData, (Float) rs.getFloat(2));
+
+            try (ResultSet rs = stmt.executeQuery("select * from " + tableName)) {
+                rs.next();
+                assertEquals(floatData, (Float) rs.getFloat(1));
+                assertEquals(floatData, (Float) rs.getFloat(2));
+            }
         }
     }
-    
+
     @BeforeAll
     public static void testSetup() throws TestAbortedException, Exception {
-        try (Connection connection = DriverManager.getConnection(connectionString);
-                Statement stmt = connection.createStatement()) {
+        try (Statement stmt = connection.createStatement()) {
             String sql1 = "create table " + tableName + " (c1 float, c2 real)";
             stmt.execute(sql1);
         }
     }
 
     @AfterAll
     public static void terminateVariation() throws SQLException {
-        try (Connection connection = DriverManager.getConnection(connectionString);
-                Statement stmt = connection.createStatement()) {
+        try (Statement stmt = connection.createStatement()) {

```

Comment:
Why are we changing the tests to use `connection` from `AbstractTest`? Dropping tables will fail if that connection is broken.

Reply:
This Static Connection is recreated everytime with every test class. This is an attempt to reduce number of connections specially when running tests with Azure databases.

Reply:
As also mentioned in PR description, this change reduces time of Junit execution by 45 secs in normal run (against SQL 2019) and much more on Azure databases.